tests/drivers/mspi/flash/testcase.yaml

```diff
@@ -37,6 +37,8 @@ tests:
     extra_configs:
       - CONFIG_MSPI_DW_HANDLE_FIFOS_IN_SYSTEM_WORKQUEUE=y
   drivers.mspi.flash.mspi_nor_no_multithreading:
+    platform_exclude:
```

Comment:
drop this and change make this

```
platform_allow: 
  - nrf54h20dk/nrf54h20/cpuapp
```
also removing all the filters


This is too much noise and filtering for a very specific usecase, everything else is covered elsewhere and testing nothread with one target should be enough.

Reply:
Done!